app-backend/db/src/main/scala/UserDao.scala

```diff
@@ -302,4 +303,15 @@ object UserDao extends Dao[User] with Sanitization {
     query
       .filter(fr"(email = $email OR personal_info ->> 'email' = $email)")
       .list
+
+  def getThinUsersForIds(ids: List[String]): ConnectionIO[List[UserThin]] =
+    ids.toNel match {
```

Comment:
This `NEL` validation logic should be pushed outward as far as possible -- handling it in the route would push all the parsing and validation to the API boundary. I know we do this elsewhere but it would be good to keep new code at least cleaner (`getThinUsersForIds` doesn't actually do anything with empty lists) and clean up as we can in other places